keymaps/intellij-idea-keymap.cson

```diff
@@ -109,17 +136,55 @@
 
   # Go to line
   'cmd-l': 'go-to-line:toggle'
+  'cmd-\\': 'unset!' # cancel the tree-view shortcut
+  'cmd-k cmd-b': 'unset!' # cancel the tree-view shortcut
```

Comment:
I would not unset the standard shortcuts as the shortcuts for idea should just be an addition to the default ones. Could you maybe remove all the `unset!` shortcuts?